lib/services.js

```diff
@@ -35,10 +44,23 @@ module.exports = function generateServices(app, ngModuleName, apiUrl) {
     { encoding: 'utf-8' }
   );
 
+  var defineLoopBackAuth = true;
+  var defineLoopBackAuthRequestInterceptor = true;
+  var defineLoopBackResource = true;
+
+  if (excludeCommonModules === "true") {
```

Comment:
Why a string? At this abstraction level, we should expect input arguments in the correct type and let callers to figure out how to convert string input into boolean values.